opt4j-optimizers/src/main/java/org/opt4j/optimizers/ea/NonDominatedFronts.java

```diff
@@ -0,0 +1,178 @@
+package org.opt4j.optimizers.ea;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import org.opt4j.core.Individual;
+import org.opt4j.core.Objective;
+import org.opt4j.core.Objectives;
+
+/**
+ * The NonDominatedFronts sorts each evaluated individual into fronts based on
```

Comment:
Please link used classes in javadoc:
`The {@link NonDominatedFronts} sorts each evaluated {@link Individual}...`